Client/src/constants/apiRequestStatus.js

```diff
@@ -0,0 +1,6 @@
+export const REQUEST_STATUSES = {
```

Comment:
Did not get structure of your project, seems you have one more client/src folders inside another Client/src. Also you have 2 folders for constants? 2 folders for components. I guess something is mixed here

Reply:
Do you remember my mistake when I start client application? We fixed it by resaving package.json, but it was temporary and arised again soon. So I decided to rename client folder to Client and mistake disappeared. I think, you have problems because of this renaming now, maybe I should replace new name of folder in some files too?

Reply:
I think it is problem not only for me :) you can check the structure in Files section for this PR, it looks like you moved just part of files to new Client folder and part of them are still under client folder, could you double check that?